server/controllers/user.js

```diff
@@ -64,4 +64,16 @@ export default class UserController {
     ]);
     return { message: "Logged out successfully." };
   };
+
+  static getUserInfo = async (req) => {
+    const userInfo = await pool.query(
+      `SELECT username, email FROM users WHERE user_id=$1;`,
+      [req.session.userId],
+    );
+    if (userInfo.rowCount) {
+      return { userInfo: userInfo.rows };
```

Comment:
maybe do it like `userInfo: userInfo.rows[0]` so that we get an object instead of an array?